DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -58,6 +60,54 @@ function PreFirstEntry()
             }
         }
         break;
+
+    case "15_Area51_Bunker":
+        if (dxr.flags.settings.starting_map < 151) { // don't delete goals if backtracking
+            player().DeleteAllGoals();
+        } else {
+            foreach AllActors(class'ElevatorMover', eMover, 'elevator_shaft') {
+                eMover.InterpolateTo(1, 0.0);
+                break;
+            }
+        }
+        break;
+
+    case "15_Area51_Final":
+        if (dxr.flags.settings.starting_map == 153) {
+            foreach AllActors(class'DeusExMover', dxMover) {
+                if (dxMover.tag == 'Page_Blastdoors' || dxMover.tag == 'door_pagearea')
+                    dxMover.InterpolateTo(1, 0.0);
+            }
+        }
```

Comment:
I would suggest a break statement here so we don't forget to add it if we extend this more (like if we add a `default:` case)
```suggestion
        }
        break;
```